cylc/flow/scripts/lint.py

```diff
@@ -44,12 +42,12 @@
 """
 
 TOMLDOC = """
-pyproject.toml configuration:{}
-   [cylc-lint]                     # any of {}
-       ignore = ['S001', 'S002']    # List of rules to ignore
-       exclude = ['etc/foo.cylc']  # List of files to ignore
-       rulesets = ['style', '728'] # Sets default rulesets to check
-       max-line-length = 130       # Max line length for linting
+pyproject.toml configuration:
+   [tool.cylc.lint]
```

Comment:
NB: there is at least one person that this will break this for.

Reply:
They will get a deprecation warning

Reply:
This isn't breaking back compat, yet.